nodejs/week1/avg.js

```diff
@@ -0,0 +1,16 @@
+const inputStrings = process.argv.splice(2);
+const inputNumbers = inputStrings.map((number) => parseFloat(number));
+const areANumbers = inputNumbers.map((number) => isNaN(number));
```

Comment:
Seems to be typo in casing of arguments
areANumbers

Reply:
Hi, Thank you so much for your review
Kindly look at the one that been updated :)